test/e2e/suite/conformance/certificates/featureset.go

```diff
@@ -83,8 +83,19 @@ const (
 	// will never pass tests that validate the duration is as expected.
 	DurationFeature Feature = "Duration"
 
-	// Wildcards denotes tests that request certificates for wildcard domains.
-	// Some issuer's disable wildcard certificate issuance, so this feature
-	// allows runs of the suite to exclude those tests that utilise wildcards.
-	Wildcards Feature = "Wildcards"
+	// WildcardsFeature denotes tests that request certificates for wildcard
+	// domains. Some issuer's disable wildcard certificate issuance, so this
+	// feature allows runs of the suite to exclude those tests that utilise
+	// wildcards.
+	WildcardsFeature Feature = "Wildcards"
+
+	// ECDSAFeature denotes whether the target issuer is able to sign
+	// certificates with an elliptic curve private key. This is useful for some
+	// issuers that have trouble being configured to support this feature.
+	ECDSAFeautre Feature = "ECDSA"
```

Comment:
AFAIK, all Issuer types should support this currently, but definitely worth keeping in 😄

Reply:
It's because of the issue with out Venafi testing environment